### DISCUSSION TITLE
A place to talk about the app and related stuff

### DISCUSSION THREAD

**Discussion by lalalasupa0** [null2264/yokai#84] [Ideas]
So... Yōkay doesn't seem to have discord or telegram, or anything similar. I think it would be cool to have either of those two, I believe it would be easier for everyone to make reports on either of those two rather than here on Github, although this wouldn't necessarily have to be the only reason for it to be done but also for the author to interact, and between everyone who uses this app, personally I love telegram more than discord, but both are cool

**Comment by null2264** [comment]
Yōkai is listed on Mihon server now on [`#j2k-yokai`](https://canary.discord.com/channels/1195734228319617024/1269521487468167178) channel: https://discord.gg/mihon
